trieste/acquisition/function.py

```diff
@@ -405,3 +408,106 @@ def probability_of_feasibility(
     mean, var = model.predict(at)
     distr = tfp.distributions.Normal(mean, tf.sqrt(var))
     return distr.cdf(tf.cast(threshold, at.dtype))
+
+
+class MonteCarloAcquisition(SingleModelAcquisitionBuilder):
```

Comment:
Can you pass a `MonteCarloAcquisition` to `EfficientGlobalOptimization` or `TrustRegion`?

Reply:
we should definitely be able to do that